src/test/java/pvs/app/entity/ProjectTest.java

```diff
@@ -0,0 +1,42 @@
+package pvs.app.entity;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import pvs.app.Application;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = Application.class)
+public class ProjectTest {
+    Project project;

```

Comment:
Suggest use a more-detailed name to this.

```suggestion
    Project testProject;
```